src/components/Avatar/Avatar.component.tsx

```diff
@@ -293,6 +303,8 @@ const Avatar: FC<AvatarProps> = ({
               />
             )}
             {effects?.vignette && <Vignette eskil={false} offset={0.5} darkness={0.5} />}
+            <BrightnessContrast brightness={0.025} contrast={0.25} />
+            <HueSaturation hue={0} saturation={-0.2} />
           </>
```

Comment:
This will simulate the old tonemapping when post postprocessing effects override the default one.